source/_components/alarm_control_panel.ialarm.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Antifurto365 iAlarm Control Panel"
+description: "Instructions how to integrate iAlarms alarms into Home Assistant."
+date: 2017-11-30 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: antifurto365-ialarm.png
+ha_category: Alarm
+ha_release: 0.60
+---
+
+The `ialarm` platform provides connectivity with the [Antifurto365](https://www.antifurtocasa365.it/) iAlarm alarm systems.
+
+This platform supports the following services: `alarm_arm_away`, `alarm_arm_home` and `alarm_disarm`.
+
+To enable this, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+alarm_control_panel:
+  - platform: ialarm
+    host: ALARM_SYSTEM_IP
+    username: YOUR_USERNAME
+    password: YOUR_PASSWORD
+```
+
+Configuration variables:
```

Comment:
Please replace this and use the configuration tag instead:
https://home-assistant.io/developers/documentation/standards/#component-and-platform-pages

Reply:
This should be fixed with aff0d53